packages/react-dom/src/client/ReactDOMComponent.js

```diff
@@ -1339,6 +1340,67 @@ export function listenToEventResponderEventTypes(
   }
 }
 
+export function createTouchHitTargetElement(
+  bottom: number,
+  left: number,
+  right: number,
+  top: number,
+  rootContainerInstance: Element | Document,
+  parentNamespace: string,
+): HTMLElement {
+  const touchHitTargetElement = ((createElement(
+    'div',
+    EMPTY_OBJECT,
+    rootContainerInstance,
+    parentNamespace,
+  ): any): HTMLElement);
+  const touchHitTargetElementStyle = touchHitTargetElement.style;
+
+  touchHitTargetElementStyle.position = 'absolute';
```

Comment:
The element should also have `zIndex = -1` to make sure it never lies over other content in the container. But like `position` this means that the container needs to have its own `zIndex` set to create a relative stacking context, otherwise this element could fall behind the container.

Reply:
This breaks the hit target for me (I can no longer press it). Let's link up on this one next week.